app/models/response_map.rb

```diff
@@ -88,4 +88,8 @@ def self.delete_mappings(mappings, force = nil)
     end
     failedCount
   end
+
+  def self.isReviewed?(team_id,assignment_id)
+    review = ResponseMap.where(reviewee_id: team_id,reviewed_object_id: assignment_id).first
```

Comment:
It will return nil if there is no record. otherwise it will return value of review(non-nil)